package.json

```diff
@@ -35,6 +35,7 @@
     "prepublish:web": "NODE_ENV=production node src/build/buildWeb.mjs",
     "prepublish:both": "npm run prepublish:web && npm run prepublish:server",
     "prepublishOnly": "npm run prepublish:web && npm run prepublish:server",
+    "bootstrap-build": "npm run build && node src/build/extractJsooMethods.js && npm run build",
```

Comment:
Our `scripts` is starting to get pretty croweded. What are thoughts on using a `Makefile` or some sort of build tool for all these building scripts?

Reply:
TBH, I prefer the package.json as a place for these - that's where a JS developer would expect them. But maybe we don't really need `prepublish:web` and `prepublish:both` as standalone scripts